dask/bytes/local.py

```diff
@@ -26,20 +25,19 @@ def __init__(self, **storage_options):
         """
         self.cwd = os.getcwd()
 
-    def _trim_filename(self, fn):
-        path = infer_storage_options(fn)['path']
+    def _normpath(self, path):
+        # ensure paths are absolute and normalized
```

Comment:
I have a slight preference for a more descriptive name here (perhaps `_normalize_path` and turning this comment into a docstring.  It's a private function, but it seems to be used frequently in public functions.  I can imagine people looking at the public functions' source code and seeing this.